LEGO1/realtime/vector.h

```diff
@@ -416,6 +408,11 @@ class Vector4 : public Vector3 {
 		m_data[3] = p_value;
 	} // vtable+0x84
 
+	float& operator[](int idx) { return m_data[idx]; }
+
+	// FUNCTION: BETA10 0x10010890
+	const float& operator[](int idx) const { return m_data[idx]; }
```

Comment:
This additional function fixes all the functions in BETA10 where it appears, and introduces no regressions in the identified BETA10 functions.